nixos/modules/services/networking/cgproxy.nix

```diff
@@ -0,0 +1,110 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
to address #208242 for this module, uses of with such as this should be avoided.
consider `inherits` where you find yourself using `lib.foo` excessively.